eogrow/utils/types.py

```diff
@@ -17,6 +17,7 @@
 BoolOrAuto = Union[Literal["auto"], bool]
 
 JsonDict = Dict[str, Any]
+RawPipelineDict = Dict[str, Any]
```

Comment:
i am not entirely happy with this name, but not sure what else it could be
its not RawConfig, because the values have already been parsed...

Reply:
Maybe `RawSchemaDict`? I can't think of any much better name. :thinking:

Reply:
Lets go with that

Reply:
Ok, renamed to `RawSchemaDict`.